src/components/header.js

```diff
@@ -37,6 +38,20 @@ const renderLevels = ({ onClickHandler, pageData, activeLevel }: Props) => {
     }
   }
 
+  if (5 > pageData.levels) {
+    const newWidth = 20 * pageData.levels
+    const newMargin = 100 - newWidth
```

Comment:
super nits: 
5, 20, and 100, could perhaps be named constants? just to make it clear what these numbers represent. the reader could currently be mistaken for thinking they were picked a little arbitrarily

Reply:
Happy to leave in an inline comment explanation here! 👍

Reply:
Done ✅